cypress/support/cypress-commands/assertion.js

```diff
@@ -553,7 +553,7 @@ Cypress.Commands.add(
           const pretext = 'Event Not Received : ';
 
           cy.log(pretext + ': expected ' + eventResponse + ' to be ' + content);
-          assert.strictEqual(eventResponse, content, pretext);
+          fireLog.strictEqual(eventResponse, content, pretext);
```

Comment:
Remove `cy.log.then()` wherever applicable. The logging would've happened internally in firelog

Reply:
Fix this in all occurrences

Reply:
Might want to double down and check again, but I am pretty sure I got all of them. I looked over it a few times, hopefully I did not miss any.